includes/tracks/events/class-wc-admin-setup-wizard-tracking.php

```diff
@@ -112,6 +111,11 @@ public function track_ready_next_steps() {
 	 * Track various events when a step is saved.
 	 */
 	public function add_step_save_events() {
+		// Always record a track on this page view.
+		if ( 'next_steps' === $this->get_current_step() ) {
+			add_action( 'admin_init', array( $this, 'track_next_steps' ), 1 );
+		}
+
```

Comment:
This should go in the switch statement a few lines below, no?

Reply:
I thought so too! But when navigating to this step, there is no `$_POST['save_step']` from the prior step happening, so the switch logic is never evaled.

Reply:
Oooh right.